### DISCUSSION TITLE
支持payjs在移动端唤起么？

### DISCUSSION THREAD

**Discussion by LiuNianJiang** [Baiyuetribe/kamiFaka#48] [新功能提交]
貌似需要使用到jsapi，请问支持唤起微信么？

**Comment by Baiyuetribe** [comment]
自己试试看，应该不支持

**Reply by LiuNianJiang** [reply]
我尝试了一下，确实是不支持的，但是我查看了PAYJS的API文档，里面可以支持JSAPI、H5类型的支付
文档：
https://help.payjs.cn/api/jsapi.html
https://help.payjs.cn/api/mweb.html

可以作为一个未来计划
Best Regards
